fe/fe-core/src/main/java/org/apache/doris/statistics/Statistics.java

```diff
@@ -149,21 +155,28 @@ public Statistics merge(Statistics statistics) {
         return this;
     }
 
-    private double computeTupleSize() {
+    private double computeTupleSize(Plan plan) {
         if (tupleSize <= 0) {
-            tupleSize = expressionToColumnStats.values().stream()
-                    .map(s -> s.avgSizeByte).reduce(0D, Double::sum);
+            tupleSize = 0;
+            for (Slot slot : plan.getOutput()) {
+                ColumnStatistic colStats = expressionToColumnStats.get(slot);
```

Comment:
why we need do this? expressionToColumnStats do not contain and only contain all slot of plan's output?

Reply:
yes，expressionToColumnStats.keySet() and plan.getOutput() are not match